### PR TITLE
BearRunning: Test exceptions

### PR DESCRIPTION
**Pull request by jayvdb**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: tests/processes/BearRunningTest.py [hunk 1]
```diff
@@ -192,6 +192,79 @@ def test_evil_bear(self):
...
+                self.global_result_dict,
+                self.message_queue,
+                self.control_queue,
+                debug=True,
+                )
+
+        self.file_name_queue.put('t')
+        self.file_dict['t'] = []
+
+        run(self.file_name_queue,
```
Comment (RaiVaibhav) [inline tests/processes/BearRunningTest.py]: Didn't get why do you calling `run`?
Same in `test_bear_impossible`

Comment (RaiVaibhav) [reply tests/processes/BearRunningTest.py]: Nvm above ^^

File: tests/processes/BearRunningTest.py [hunk 2]
```diff
@@ -192,6 +192,79 @@ def test_evil_bear(self):
...
+                self.global_bear_queue,
+                self.file_dict,
+                self.local_result_dict,
+                self.global_result_dict,
+                self.message_queue,
+                self.control_queue,
+                debug=True,
+                )
+
+        self.file_name_queue.put('t')
```
Comment (RaiVaibhav) [inline tests/processes/BearRunningTest.py]: code repetition here, no need to reassign the same value

File: tests/processes/BearRunningTest.py [hunk 3]
```diff
@@ -192,6 +192,79 @@ def test_evil_bear(self):
...
+        self.file_dict['t'] = []
```
Comment (RaiVaibhav) [inline tests/processes/BearRunningTest.py]: same ^^

**Comment by jayvdb** [comment]
ack cc94fd6 ecfac51

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**